setup.py

```diff
@@ -15,7 +15,7 @@
 if sys.argv[-1] == 'publish':
     if sys.version_info.major < 3:
         print('Error: do not publish release with python2!')
-        sys.exit(1)
+        sys.exit(1)    
```

Comment:
I think there's a few extra spaces at the end of this line which shouldn't be here